include/boxed-cpp/boxed.hpp

```diff
@@ -164,11 +164,17 @@ constexpr auto unbox(boxed::boxed<From, FromTag> const& from) noexcept
     return static_cast<To>(from.value);
 }
 
+template <typename T>
+concept con_boxed = requires(T t)
```

Comment:
I try not to nitpick, but do we have any better name than `con_boxed` maybe? Worst case, just spell it out, like `concept_boxed` or `boxed_concept`? :)

Reply:
While I am all in favor of concepts (finally we have them!), I wonder if that might induce problems on Contour's CI side, since we still need to support older compilers (or do we?) Maybe the problem is gone, since even for Ubuntu 18.04 build (probably the oldest distro) we're using a custom way more recent Clang, IIRC?